src/_P004_Dallas.ino

```diff
@@ -224,211 +323,234 @@ byte Plugin_004_DS_scan(byte getDeviceROM, uint8_t* ROM)
 *   11 bits resolution -> 375 ms
 *   12 bits resolution -> 750 ms
 \*********************************************************************************************/
-void Plugin_004_DS_startConversion(uint8_t ROM[8])
+void Plugin_004_DS_startConversion(uint8_t * ROM,uint8_t res, boolean parasite)
 {
-    Plugin_004_DS_reset();
-    Plugin_004_DS_write(0x55); // Choose ROM
-    for (byte i = 0; i < 8; i++)
-        Plugin_004_DS_write(ROM[i]);
-    Plugin_004_DS_write(0x44); // Take temperature mesurement
+        if(Plugin_004_DS_reset()) {
+                Plugin_004_ROM_select(ROM);
+                Plugin_004_DS_write(STARTCONVO);
+                if(parasite) {
+                        digitalWrite(Plugin_004_DallasPin, HIGH);
+                        pinMode(Plugin_004_DallasPin, OUTPUT);
+
+                        switch (res)
+                        {
+                        case 9:
+                                delay(94);
+                                break;
+                        case 10:
+                                delay(188);
+                                break;
+                        case 11:
+                                delay(376);
+                                break;
+                        default:
+                                delay(751);
```

Comment:
This is also using astronomical long delays.

Reply:
Should be a setting or a completely new plugin.